1 explicit pages:
  config/english/config_en.prc

2 implicit pages:
  /c/Panda3D-1.8.1/etc/Config.prc
  /c/Panda3D-1.8.1/etc/Confauto.prc

serverVersion:  sv-1.0.0
PAccount: accountServer from launcher:  
PAccount: default accountServer:  https://pirates.go.com
Traceback (most recent call last):
  File "C:\Panda3D-1.8.1\python\lib\runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "C:\Panda3D-1.8.1\python\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "C:\Users\Tyler\Desktop\POTCOR-src\pirates\piratesbase\PiratesStart.py", line 61, in <module>
    from pirates.distributed import PiratesClientRepository
  File "pirates\distributed\PiratesClientRepository.py", line 57, in <module>
    from pirates.login.AvatarChooser import AvatarChooser
  File "pirates\login\AvatarChooser.py", line 31, in <module>
    from pirates.pirate import Pirate
  File "pirates\pirate\Pirate.py", line 2, in <module>
    from Human import Human
  File "pirates\pirate\Human.py", line 17, in <module>
    from pirates.makeapirate import PirateMale
  File "pirates\makeapirate\PirateMale.py", line 13, in <module>
    from pirates.inventory import ItemGlobals, DropGlobals
  File "pirates\inventory\DropGlobals.py", line 9, in <module>
    from pirates.battle import EnemyGlobals
  File "pirates\battle\EnemyGlobals.py", line 10, in <module>
    realAvatarTypes = AvatarTypes()
TypeError: 'module' object is not callable
